node/config.go

```diff
@@ -108,6 +108,21 @@ type Config struct {
 	DispersalAuthenticationKeyCacheSize int
 	// the timeout for disperser keys (after which the disperser key is reloaded from the chain)
 	DisperserKeyTimeout time.Duration
+
+	// The size of the thread pool used to download chunks from the network is computed by the following formula:
+	// (number of CPUs) * DownloadPoolMultiplier + DownloadPoolConstant
+	DownloadPoolMultiplier int
+
+	// The size of the thread pool used to download chunks from the network is computed by the following formula:
+	// (number of CPUs) * DownloadPoolMultiplier + DownloadPoolConstant
+	DownloadPoolConstant int
+
+	// If true, use littDB instead of levelDB for v2 storage. Note than in its current form, no data migration is
```

Comment:
> Note than in its current form, no data migration is performed when this setting is enabled.

Is this true? Seems there are migration considerations in `validator_store.go`

Reply:
Reduced comment to `// If true, use littDB instead of levelDB for v2 storage.`. When I originally wrote this comment, I hadn't yet implemented migration. Once I did add migration, I forgot to circle back.